pkg/intermediate/aggregate.go

```diff
@@ -542,6 +543,14 @@ func (a *AggregationProcess) aggregateRecords(incomingRecord, existingRecord ent
 					incomingVal := ieWithValue.GetStringValue()
 					existingIeWithValue.SetStringValue(incomingVal)
 				}
+			case "httpVals":
+				incomingVal := ieWithValue.GetStringValue()
+				existingVal := existingIeWithValue.GetStringValue()
+				updatedHttpVals, err := fillHttpVals(incomingVal, existingVal)
+				if err != nil {
+					return fmt.Errorf("httpVals could not be updated: %v", err)
```

Comment:
Do you think we should print the error log instead? Or we should reset fields we changed above. Otherwise the existing connection might have wrong info comes from the incoming connection.

Reply:
That is a good idea, allow me to change it